crates/ibc/src/applications/transfer/context.rs

```diff
@@ -444,6 +441,7 @@ pub(crate) mod test {
         let counterparty = Counterparty::new(port_id.clone(), Some(channel_id.clone()));
 
         (
+            MockTokenTransferModule,
             ctx,
```

Comment:
`ctx` not used anymore. Can entirely be removed.

Reply:
[34f658d](https://github.com/cosmos/ibc-rs/pull/787/commits/34f658db4df71861a4af853e611abf0bf78f8f0a)